v2-migration/src/main/java/software/amazon/awssdk/v2migration/TransferManagerMethodsToV2.java

```diff
@@ -47,6 +50,19 @@ public class TransferManagerMethodsToV2 extends Recipe {
     private static final MethodMatcher COPY_BUCKET_KEY =
         v2TmMethodMatcher("copy(String, String, String, String");
 
+    private static final MethodMatcher DOWNLOAD_DIR = v2TmMethodMatcher("downloadDirectory(String, String, java.io.File)");
+
+    private static final MethodMatcher RESUME_DOWNLOAD = v2TmMethodMatcher("resumeDownload(..)");
+    private static final MethodMatcher RESUME_UPLOAD = v2TmMethodMatcher("resumeUpload(..)");
+    private static final MethodMatcher SHUT_DOWN_NOW = v2TmMethodMatcher("shutdownNow()");
+
+
+
+    private static final Pattern S3_TM_CREDENTIAL = Pattern.compile(V2_TM_CLIENT);
```

Comment:
```suggestion
    private static final MethodMatcher SHUT_DOWN_NOW = v2TmMethodMatcher("shutdownNow()");

    private static final Pattern S3_TM_CREDENTIAL = Pattern.compile(V2_TM_CLIENT);
```

nit: remove extra whitespace lines